trace_viewer/extras/about_tracing/profiling_view.html

```diff
@@ -402,6 +402,34 @@
       this.uploadOverlay_ = null;
     },
 
+    requestFilename_: function() {
+
+      // unsafe filename patterns:
+      var illegalRe = /[\/\?<>\\:\*\|":]/g;
+      var controlRe = /[\x00-\x1f\x80-\x9f]/g;
+      var reservedRe = /^\.+$/;
+
+      var filename;
+      var defaultName = this.activeTrace_.filename;
+      var custom = prompt('Filename? (.json appended) Or leave blank:');
+      if (custom === null)
+        return false;
+
+      if (custom) {
+        filename = defaultName.replace(/\.json$/, ' ' + custom) + '.json';
```

Comment:
I'm keeping the "trace" of `this.activeTrace_.filename` intact and adding a space after it.